joern-cli/frontends/jssrc2cpg/src/main/scala/io/joern/jssrc2cpg/astcreation/AstForTemplateDomCreator.scala

```diff
@@ -36,7 +36,13 @@ trait AstForTemplateDomCreator { this: AstCreator =>
   }
 
   protected def astForJsxAttribute(jsxAttr: BabelNodeInfo): Ast = {
-    val domNode = createTemplateDomNode(jsxAttr.node.toString, jsxAttr.code, jsxAttr.lineNumber, jsxAttr.columnNumber)
+    val colon = pos(jsxAttr.json)
```

Comment:
I think this could use a comment explaining what's going on, but it's your code so it's up to you.

Reply:
Done.